ome_zarr/cli.py

```diff
@@ -99,6 +106,12 @@ def main(args: List[str] = None) -> None:
     parser_info.add_argument("--stats", action="store_true")
     parser_info.set_defaults(func=info)
 
+    # validate
+    parser_validate = subparsers.add_parser("validate")
+    parser_validate.add_argument("path")
+    parser_validate.add_argument("--warnings", action="store_true")
```

Comment:
yes. We have various terminology: There are Spec rules that `SHOULD` be followed (so it's `recommended` that you do). These are covered by a `strict schema` and if you don't then this gives you `warnings` (if you use that flag)!
It would be nice to settle on ONE term that could cover all these cases?

Reply:
Agreed. Trying to look into the terminology used by others, I came back to SHACL which we have not decided to adopt at this stage but we might want to come back to in the near future.

A relevant concept is the idea of severity defined [here](https://www.w3.org/TR/shacl/#results-severity) and used by the [validation report](https://book.validatingrdf.com/bookHtml011.html#sec127). In particular, looking at the [pySHACL](https://github.com/RDFLib/pySHACL) library, the `--allow-warnings` option effectively controls the severity level beyond which the result will be considered as invalid.